vagrant/ansible/roles/samba.setup/tasks/cephfs/main.yml

```diff
@@ -4,7 +4,29 @@
     name: samba
     state: present
 
+- name: Prepare CephFS VFS support
+  when: config.be.variant == 'vfs'
+  block:
+    - name: Install CephFS VFS module
+      yum:
+        name: samba-vfs-cephfs
+        state: present
+
+    - name: Set selinux in permissive mode
+      selinux:
+        policy: targeted
+        state: permissive
+
+    - name: Skip check of share path
+      lineinfile:
+        path: /usr/share/ctdb/events/legacy/50.samba.options
```

Comment:
Proper location would be _/etc/ctdb/events/legacy/50.samba.options_.

Reply:
Done